### PR TITLE
Update st2rc.sample.ini

### PR DESCRIPTION
**Pull request by punkrokk**
I've added in the [stream] stanza and examples for remote st2client functionality. 
Please let me know anissue should be created. 

I couldn’t use the —tail option without the stream stanza

Fixes: #4684 

I'm not sure if this page needs to be updated - https://github.com/StackStorm/st2/tree/master/st2client

I don't know what a reactor and some other terms I'm not aware of are there, but it may make sense. cc @LindsayHill

### CODE REVIEW

File: conf/st2rc.sample.ini [hunk 1]
```diff
@@ -22,6 +22,15 @@ password = testpassword
 
 [api]
 url = http://127.0.0.1:9101/v1
+# or from remote
+# url = http://127.0.0.1/api/v1
```
Comment (Kami) [inline conf/st2rc.sample.ini]: I assume here it should be some other external domain / IP and not, 127.0.0.1, right?

Perhaps we should use example.com?

**Comment by Kami** [comment]
Thanks for the contribution :+1: 

I updated the code to use ``example.com`` for remote deployment and also updated the URLs to use correct paths.

File: conf/st2rc.sample.ini [hunk 2]
```diff
@@ -22,6 +22,15 @@ password = testpassword
 
 [api]
 url = http://127.0.0.1:9101/v1
+# or for a remote instance
+# url = https://example.com/api/v1
 
 [auth]
 url = http://127.0.0.1:9100/
+# or for a remote instance
```
Comment (Kami) [inline conf/st2rc.sample.ini]: I'll actually confirm it works correctly with the paths I used.

it's a bit annoying those URLs are not consistent, but making them consistent might require code changes :)

Comment (Kami) [reply conf/st2rc.sample.ini]: Confirmed those URLs work correctly with the default nginx config so the same consistent suffix can be used everywhere.

**Comment by punkrokk** [comment]
Thanks all. Good call on example.com. 

Any thoughts about the reactor stuff? Is that legacy terminology?

**Comment by Kami** [comment]
I'm not sure what you mean / refer to with reactor.

**Comment by punkrokk** [comment]
Never mind. I saw it somewhere. But can’t find it again.

**Comment by punkrokk** [comment]
Basically I saw a reactor referenced in the read me and I had no idea what it was

**Comment by Kami** [comment]
@punkrokk Ah, I see.

It looks like readme is out of date and needs some cleanup :)

**Pull request merged**